src/styles/main.scss

```diff
@@ -1,3 +1,61 @@
+$day-size: 100px;
+$day-gap: 1px;
+$days-names: mon 0, tue 1, wed 2, thu 3, fri 4, sat 5, sun 6;
+
+
 body {
   margin: 0;
 }
+
+.calendar {
+  max-width: 7 * $day-size + 10px;
+  max-height: 5 * $day-size + 6 * $day-gap;
+  gap: $day-gap;
+  display: flex;
+  flex-wrap: wrap;
+
+  @each $value, $nums in $days-names {
+    &--start-day--#{$value} {
+      & :first-child {
+        margin-left: $nums * $day-size + $nums * $day-gap;
+      }
+    }
+  }
+  &__day:hover {
+    cursor: pointer;
+    background-color: pink;
+    transform: translate(0,-20px);
+  }
+  &__day {
+    height: $day-size;
+    width: $day-size;
+    background-color: #eee;
+    box-sizing: border-box;
+    display: flex;
+    border: 1px solid black;
```

Comment:
1. Fix everywhere
<img width="571" alt="image" src="https://github.com/mate-academy/layout_calendar/assets/49764881/f709fc60-76fb-47a1-a3cd-5cfcc7b1504c">

2. 
```
* {
  box-sizing: border-box;
}
```

3. Lets add transition property, but be attentive to checklist
<img width="1009" alt="image" src="https://github.com/mate-academy/layout_calendar/assets/49764881/981a1590-bc62-405c-ad8f-a425aa2fcf00">